cli/cmd/cluster-apply.go

```diff
@@ -111,7 +111,11 @@ func clusterApply(contextLogger *log.Entry, options clusterApplyOptions) error {
 
 	fmt.Printf("\nYour configurations are stored in %s\n", c.assetDir)
 
-	kubeconfig, err := getKubeconfig(contextLogger, c.lokomotiveConfig, true)
+	kg := kubeconfigGetter{
```

Comment:
I'm not super convinced by this getter (especially regarding naming) but I think this is an improvement over the current state so I'm not blocking this.

Reply:
Yeah, once moved to separate package, we need to continue refactoring...